### ISSUE TITLE
Scalling down with FFmpegFrameGrabber produces low quality results 

### ISSUE THREAD

**Issue by b005t3r** [bytedeco/javacv#1279]
This is something I noticed. I'm not actually sure if this is something you can fix on your end, I just want to make sure my setup is OK.

So, I decode frames from a video. Video resolution is 1920x1080, I need 1280x720.

Now, I can scale it down with `FFmpegFrameGrabber`, like so:
```java
            grabber.setImageWidth(1280);
            grabber.setImageHeight(720);
            grabber.setPixelFormat(PIXEL_FORMAT);
            grabber.setImageScalingFlags(SWS_BICUBIC);
```

or I can decode at video's resolution and scale it down with OpenCV.

The thing is get better results with the OpenCV method, so I wanted to make sure if there's anything I could do to up the quality when using `FFmpegFrameGrabber` alone? I tried different image scaling flags, they don't seem to make much of a difference (if any at all).

**Comment by saudet** [comment]
It's possible OpenCV is better, yes, I've never compared them, but there's a few more you can try for FFmpeg: https://github.com/bytedeco/javacpp-presets/blob/master/ffmpeg/src/gen/java/org/bytedeco/ffmpeg/global/swscale.java#L76-L86

**Comment by b005t3r** [comment]
Yes, I've seen those and tried every single one - there's very little difference.

**Comment by saudet** [comment]
FWIW, you'll probably get more help concerning that if you ask the developers of FFmpeg.

**Comment by YukiDmZhao** [comment]
set PIXEL_FORMAT   value  from 1 to n , can not run .

**Comment by YukiDmZhao** [comment]
grabber.setPixelFormat(PIXEL_FORMAT);  this  PIXEL_FORMAT  value???

**Comment by b005t3r** [comment]
It's defined like that:

```java
    public static final int PIXEL_FORMAT = AV_PIX_FMT_BGR24;
```
